### DISCUSSION TITLE
为什么网络正常，但是 Steam 下载 显示 无连接？

### DISCUSSION THREAD

**Discussion by pjy612** [pjy612/SteamManifestCache#11] [Question / Answer]
为什么网络正常，但是 Steam 下载 显示 无连接？

**Comment by pjy612** [comment]
无连接 一般有三种情况。
1. 真的网络不稳定 连不上Steam 了 或者 Steam 服务器挂了。
2. 使用了 加速器，在 加速器工作时 对加速节点进行了切换。
3. 假入库的游戏因为清单或密钥不全，从而失败过多，导致与Steam服务器的连接通道耗尽。

小知识：
Steam 对自家服务器的访问 有 节点（连接通道） 的概念。【通俗点能形容成一个“外卖站”
Steam 登陆成功后 会获取 一批 节点（连接通道）。【招一群外卖小哥
Steam 下载时 尝试获取密钥或清单 会消耗 节点。【让小哥去收送快递外卖
当然 账号 数据齐全 权限够的时候，成功获取到清单数据，则节点会被回收。【地址正确，小哥资质齐全，会成功返回。
当数据不全或权限不足 导致失败时，节点直接被放弃。【地址不对，小哥迷路。资质不全，小哥被扣押
Steam 每隔一段时间 会刷新 节点（连接通道）列表。【隔一段时间会重新招一群小哥

所以 推荐做法是 ，将 入库的游戏放入 Steam 下载列表 的 `未安排` 队列。
并且 对没有的游戏（假入库，未入正，非共享） 调整为 `启动时再检查更新`
把 Steam 允许 `自动更新` 的时间范围 改成 **不常用的时间** 比如 凌晨 3-4 点

不然 Steam 会 各种尝试对 `已安排` 的队列进行下载，失败越多越容易 无连接 
甚至 已经在下载途中，也会因为其他游戏 `失败的更新检测` 导致 连接耗光 中断下载

PS.
加速器有时候会导致 下载流量 走加速节点，从而下载速度慢。此时关了加速重试下载一般会更快。

**Comment by guozqwin** [comment]
大佬，现在steamtools是玩不了了吗，任何游戏打开都是无网络连接
